R/install_reqs.R

```diff
@@ -162,27 +164,28 @@ install_reqs = function(reqs, dryrun, verbose = dryrun,
   #' @param ... values to pass to get_installed
   #' @return data.frame of installed packages and their versions
 
-  INSTALL = 'Installing %s %s'
-  INSTALL_VERSION = 'at version %s'
-  NOT_INSTALLED = 'NOTE: %s is not currently installed.'
-  BAD_VERSION = 'NOTE: %s is current installed at version %s which is not sufficient.'
-  NONE_EXISTS = 'ERROR: No version exists for %s which meets requirements.'
-  OTHER_FAIL = 'ERROR: Requirement %s could not be satisified for some reason.'
+  INSTALL = "Installing %s %s"
+  INSTALL_VERSION = "at version %s"
+  NOT_INSTALLED = "NOTE: %s is not currently installed."
+  BAD_VERSION = "NOTE: %s is current installed at version %s which is not sufficient."
+  NONE_EXISTS = "ERROR: No version exists for %s which meets requirements."
+  OTHER_FAIL = "ERROR: Requirement %s could not be satisified for some reason."
 
   installed = get_installed(...)
 
   failures = 0
 
   # Install git
   failures = failures +
-    install_special_req(reqs$git, '^git\\+', devtools::install_git, dryrun, verbose) +
-    install_special_req(reqs$svn, '^svn\\+', devtools::install_svn, dryrun, verbose) +
-    install_special_req(reqs$bioc, '^bioc\\+', devtools::install_bioc, dryrun, verbose) +
-    install_special_req(reqs$url, '', devtools::install_url, dryrun, verbose) +
-    install_special_req(reqs$local, '', install_local, dryrun, verbose) +
-    install_unversioned(reqs$unversioned, dryrun, verbose, repo) +
-    install_versioned(reqs$versioned, dryrun, verbose, repo)
-
-  if(failures > 0) stop(paste0('There were ',failures,' package(s) which could not be installed.'))
+    install_special_req(reqs$git, "^git\\+", devtools::install_git, dryrun, verbose) +
+    install_special_req(reqs$svn, "^svn\\+", devtools::install_svn, dryrun, verbose) +
+    install_special_req(reqs$bioc, "^bioc\\+", devtools::install_bioc, dryrun, verbose) +
+    install_special_req(reqs$url, "", devtools::install_url, dryrun, verbose) +
+    install_special_req(reqs$local, "", install_local, dryrun, verbose) +
+    install_unversioned(reqs$unversioned, installed, dryrun, verbose, repo) +
```

Comment:
This is the only relevant change for actual review.

`install_unversioned()` & `install_versioned()` were being called with what looked to be invalid arguments

redundant inline diff for redundancy

pre-changed code
```r
install_unversioned(reqs$unversioned, dryrun, verbose, repo) +
install_versioned(reqs$versioned, dryrun, verbose, repo)
```

post change
```r
install_unversioned(reqs$unversioned, installed, dryrun, verbose, repo) +
install_versioned(reqs$versioned, installed, dryrun, verbose, repo)
```